crates/extensions/c8y_mapper_ext/src/tests.rs

```diff
@@ -2948,6 +2992,7 @@ pub(crate) async fn spawn_c8y_mapper_actor(
     SimpleMessageBox<C8YRestRequest, C8YRestResult>,
     SimpleMessageBox<NoMessage, FsWatchEvent>,
     SimpleMessageBox<SyncStart, SyncComplete>,
+    SimpleMessageBox<IdUploadRequest, IdUploadResult>,
```

Comment:
Not for this PR, but seeing a million tests getting updated because of this addition makes me feel that we should have just returned a struct here instead of a tuple.